src/test/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/SupplierSyncMessageCornellTest.java

```diff
@@ -2,81 +2,65 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.util.stream.Stream;
 
-import org.apache.commons.io.FileUtils;
-import org.apache.logging.log4j.Level;
-import org.apache.logging.log4j.core.config.Configurator;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
-import org.mockito.Mockito;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.api.parallel.Execution;
 import org.xml.sax.SAXException;
 
-import edu.cornell.kfs.module.purap.CUPurapParameterConstants;
-import edu.cornell.kfs.module.purap.CuPurapTestConstants;
-import edu.cornell.kfs.module.purap.batch.JaggaerGenerateSupplierXmlStep;
-import edu.cornell.kfs.sys.service.CUMarshalService;
-import edu.cornell.kfs.sys.service.impl.CUMarshalServiceImpl;
 import edu.cornell.kfs.sys.util.CuXMLUnitTestUtils;
 import jakarta.xml.bind.JAXBException;
 
-public class SupplierSyncMessageTest {
+@Execution(ExecutionMode.SAME_THREAD)
+public class SupplierSyncMessageCornellTest extends SupplierSyncMessageTestBase{
     private static final String US_DOLLAR_CURRENCY_CODE = "usd";
 
     private static final String F_FALSE = "F";
 
     private static final String T_TRUE = "T";
 
-    private static final String INPUT_FILE_PATH = "src/test/resources/edu/cornell/kfs/module/purap/jaggaer/xml/";
-    private static final String OUTPUT_FILE_PATH = INPUT_FILE_PATH + "outputtemp/";
-    private static final String BASIC_FILE_EXAMPLE = "SupplierSyncMessageBasic.xml";
-
-    private File outputFileDirectory;
-
-    private CUMarshalService marshalService;
-
-    @BeforeEach
-    void setUpBeforeClass() throws Exception {
-        Configurator.setLevel(CUMarshalServiceImpl.class, Level.DEBUG);
-        marshalService = new CUMarshalServiceImpl();
-        outputFileDirectory = new File(OUTPUT_FILE_PATH);
-        outputFileDirectory.mkdir();
-    }
-
-    @AfterEach
-    void tearDownAfterClass() throws Exception {
-        marshalService = null;
-        FileUtils.deleteDirectory(outputFileDirectory);
+    private static final String REQUEST_FILE_EXAMPLE = "SupplierSyncMessage-RequestMessage-CornellTestData.xml";
+    private static final String RESPONSE_GOOD_FILE_EXAMPLE = "SupplierSyncMessage-ResponseMessage-CornellTestData-response-good.xml";
+    private static final String RESPONSE_BAD_FILE_EXAMPLE = "SupplierSyncMessage-ResponseMessage-CornellTestData-response-bad.xml";
+    
+    @Override
+    protected  String buildOutputFilePath() {
+        return INPUT_FILE_PATH + "outputtemp/";
```

Comment:
We have various unit tests that use a "test" directory under the project's root for data output. It would be best if we could standardize on a location to place the output files from our various tests.

If you think it's not practical to change your tests' output paths at this time, then please create a follow-up user story for further analysis and remediation.

Reply:
ok